### PR TITLE
coafile: Check all files

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

**Comment by sils** [comment]
Comment on 7f27c27aab55cfbf8f58ac1b5a4b1ee9bc682f5d, file README.md, line 1.

The text does not comply to the set style.

*MarkdownBear, severity NORMAL, section `markdown`.*

The issue can be fixed by applying the following patch:

```diff
--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
-#coala-landing
+\#coala-landing
 
-###The coala.io landing page of the future! (This is a WIP, the "website" repo is the current one.)
+\###The coala.io landing page of the future! (This is a WIP, the "website" repo is the current one.)
 
 Get up and running on local machine
 
```

**Comment by sils** [comment]
Comment on 7f27c27aab55cfbf8f58ac1b5a4b1ee9bc682f5d, file README.md, line 12.

The text does not comply to the set style.

*MarkdownBear, severity NORMAL, section `markdown`.*

The issue can be fixed by applying the following patch:

```diff
--- a/README.md
+++ b/README.md
@@ -9,4 +9,5 @@
 $ cd landing
 $ python -m SimpleHTTPServer 8000
 ```
-It's now running on http://localhost:8000/+
+It's now running on <http://localhost:8000/>
```

**Comment by sils** [comment]
Comment on 7f27c27aab55cfbf8f58ac1b5a4b1ee9bc682f5d, file resources/css/style.css, line 220.

Line contains following spacing inconsistencies:
- No newline at EOF.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/resources/css/style.css
+++ b/resources/css/style.css
@@ -217,4 +217,4 @@
   line-height:50px;
   color:lightslategrey !important;
   background-color:#fff !important
-}+}
```

**Comment by sils** [comment]
Comment on 7f27c27aab55cfbf8f58ac1b5a4b1ee9bc682f5d, file resources/css/style.css, line 9.

Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/resources/css/style.css
+++ b/resources/css/style.css
@@ -6,7 +6,7 @@
   height: 48px;
   padding: 0;
   margin: 0;
-  
+
 }
 
 .tabs .tab a {
```

**Comment by sils** [comment]
Comment on 7f27c27aab55cfbf8f58ac1b5a4b1ee9bc682f5d, file README.md, line 12.

Line contains following spacing inconsistencies:
- No newline at EOF.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/README.md
+++ b/README.md
@@ -9,4 +9,4 @@
 $ cd landing
 $ python -m SimpleHTTPServer 8000
 ```
-It's now running on http://localhost:8000/+It's now running on http://localhost:8000/
```

File: README.md [hunk 1]
```diff
@@ -9,4 +9,5 @@ $ git clone https://github.com/coala/landing.git
 $ cd landing
 $ python -m SimpleHTTPServer 8000
 ```
-It's now running on http://localhost:8000/
\ No newline at end of file
+
+It's now running on <http://localhost:8000/>
```
Comment (meetmangukiya) [inline README.md]: As this is the output, I think we should Include in the code-block as well ?

Comment (hemangsk) [inline README.md]: this is not the output actually

Comment (sils) [inline README.md]: this is out of scope for the commit, also it doesn't look like it's the output of that command ut an explanation

**Comment by hemangsk** [comment]
ack #8

**Comment by hemangsk** [comment]
ack 1c5b30217ed20e40d410b450ac216da0bb3a1f35

**Comment by hemangsk** [comment]
ack 28a97c4950d26b614afd45a23b54c9bbb05c940f

**Comment by hemangsk** [comment]
@rultor merge
